common/shlibs

```diff
@@ -1653,6 +1653,9 @@ libosmoctrl.so.0 libosmocore-0.7.0_1
 libgtkglext-x11-1.0.so.0 gtkglext-1.2.0_4
 libgdkglext-x11-1.0.so.0 gtkglext-1.2.0_4
 libXaw3d.so.8 libXaw3d-1.6.2_1
+libXaw3dxft.so.8 libXaw3dXft-1.6.2h_1
+libshiboken2.so.5.15 libshiboken2-5.15.0_1
+libpyside2.so.5.15 libpyside2-python3-5.15.0_1
```

Comment:
there seem to be two wrong/additional shlib entries.

Reply:
How idea how that happened...